extensions/resteasy-reactive/quarkus-resteasy-reactive-jackson/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -14,3 +14,4 @@ metadata:
   - "web"
   - "reactive"
   status: "experimental"
+description: "Jackson integration for RESTEasy Reactive. This extension is not compatible with the quarkus-resteasy extension, or any of the extensions that depend on it."
```

Comment:
So... for all the other extensions, it was decided to add the description to the runtime pom.xml. (my personal preference would have been to do it as you did but it was decided differently)

Also aren't the description far too long for code.quarkus.io? @ia3andy ?

Reply:
I really don't care TBH :).

Up to you'all to decide

Reply:
It's ok for code.quarkus.io, it's truncated and you have tooltip to show the long version.

Reply:
I think it would be better to have it in the pom.xml to have it displayed in the different 
maven repositories, but @maxandersen seemed to want it here.. any reason?

Reply:
It was @maxandersen who fought me for having them in the pom.xml :]

Reply:
Yes, this information should be in the `pom.xml`. I am not 100% sure butt think it is ignored in this YAML file

Reply:
No gsmet. I fought that it should be necessary to duplicate information, minimal maintance/setup for extension writers and that the info in Pom.xml is used on other sites too (think search.maven.org). 

Thus we said that the Pom.xml is the default fallback so only those who want to maintain duplicate text blocks or like in this case add some additional info are burdened with it.

Reply:
@geoand let's put it in the pom.xml I think this was just a misunderstanding in the end..

Reply:
I suggested to put it here as I assumed it would only really make sense for code quarkus.

Reply:
Whatever y'all think is best I'll do :)

Reply:
@maxandersen well it's also relevant for maven, I don't see why it wouldn't be?

Reply:
I think we need the basic information in the pom and the whole stuff with incompatibilities in the yaml.

IIRC, the Yaml has precedence over the pom.xml. That way the incompatibility sentence will only appear for code.quarkus.io and we would have some info in Maven anyway.

Does it make sense?